kmdf/queue.c

```diff
@@ -298,6 +299,46 @@ ECTestQueueInitialize(
     return status;
 }
 
+/*
+ * Function: NTSTATUS FfaDrvTestDirectCall
+ *
+ * Description:
+ * Test function to directly make FFA call through the ffadrv interface rather than through ACPI
+ *
+ * Parameters:
+ * None
+ *
+ * Return Value:
+ * Returns an NTSTATUS value indicating the success or failure of FFA command
+ * If
+ */
+NTSTATUS
+FfaDrvTestDirectCall(VOID)
+{
+  NTSTATUS status = STATUS_SUCCESS;

```

Comment:
>   [](http://example.com/codeflow?start=0&length=2)

nit: it looks like the rest of this file uses 4-space indentation; consider aligning with that in this function